### DISCUSSION TITLE
Regarding the issue of scrollbar in dark mode

### DISCUSSION THREAD

**Discussion by Back-T0** [sumatrapdfreader/sumatrapdf#3839] [Ideas]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/78223825/21c8d6d5-b89d-4843-a520-3f039ce0b972)

It's not a big deal, but it's best if the author can optimize it

**Comment by Back-T0** [comment]
And it would be nice if the dark mode could follow the system

**Reply by GitHubRulesOK** [reply]
the scrollbar Is part of Microsoft Visual C Programming (IE it is a WINDOWS scrollbar)
It can be changed same as  all other windows by using MS Windows Controls or a 3rd party theming application such as Windows Blinds

So SumatraPDF has not changed that Windows feature it is NOT A NEW ISSUE (covered by many requests related to themes)
It is down to Microsoft Windows ensuring Theming is easily done per application but for compatibility all apps use the same scrollbars, and thus an app should NOT change that behaviour without changing ALL other applications windows to match (I.E. windows blinds).

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/73d9b3e5-e238-4bd9-8cc6-76109136d47e)

A major Hurdle is EVEN WINDOWS on its own has problems maintaining a common style across all the apps over its timespan of using Dark scrollbars as seen by setting scrollbars to dark in Windows Theming HENCE use a 3rd Party app like Windows Blinds..

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/f5661466-06c3-41ce-93aa-bef098c7b107)

**Reply by Back-T0** [reply]
I see what you mean, thank you for your patience. 

Another question is, I've found that when I use the scroll wheel on the right scroll bar, I scroll half a page, but the bottom scroll bar (which appears when the document is zoomed in) doesn't respond when I use the scroll wheel, is it worth adding left and right scrolling to the bottom scrollbar?

I don't want to worry about scrollbars, but I use them a lot and want to make Sumatra better

**Comment by kjk** [comment]
You can't easily change the color of scrollbars and implementing custom scrollbars is a lot of work. I'm not saying no but it's a low priority.

**Reply by Back-T0** [reply]
Thank you for your reply, I know its priority.

Another question is, does Sumatra dark mode follow the system? I mean when I use the system Dark Theme Sumatra is able to detect and automatically convert the dark mode.

By the way, Sumatra is really superior and I can't do without it anymore, it wouldn't have been possible without your genius code editors.

**Reply by lokmac** [reply]
I'm not sure if this has changed in the past 2 years, and I'm not sure if I understand correctly (and don't understand Windows app dev at all), but the scroll bar is not following the Windows theme to be dark. It is permanently in Light Mode regardless of the standard scrollbar appearance in Windows. Just FYI.

**Comment by GitHubRulesOK** [comment]
just examples of how **extra 3rd party apps** can help to improve your windows experience but they require system wide access.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4a967ee2-60d9-4ee9-9a0c-dd28e7d136d2)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/1ede906d-b46f-4c4e-99cc-4701b245437e)

**Reply by Back-T0** [reply]
I got it, thank you

**Comment by Back-T0** [comment]
Another strange phenomenon is that hiding the scrollbar can cause stuttering

https://github.com/sumatrapdfreader/sumatrapdf/assets/78223825/fd27b104-7685-408a-af1d-6455d4ace537

**Reply by GitHubRulesOK** [reply]
@Back-T0 
that's ODD 
for me with a small or big page, high zoom to force need for scrollbars, slow 32bit machine cheap mouse (has other problems outside SumatraPDF)
scrollbars on or off does not seem to make any difference
